### PR TITLE
21173 add meta_data field to batch_processing table

### PR DESCRIPTION
**Pull request by AimeeGao**
*Issue #:* /bcgov/entity#21173

*Description of changes:*

- Add meta_data field to batch_processing table to alembic script. Field type is jsonb.
- Add meta_data field to code model
- Update tests as req’d

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/migrations/versions/ef8f033d317b_add_meta_data_to_batch_processing.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# revision identifiers, used by Alembic.
+revision = 'ef8f033d317b'
+down_revision = '508c7466b2af'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('batch_processing', sa.Column('meta_data', JSONB, nullable=True))
```
Comment (chenhongjing) [inline legal-api/migrations/versions/ef8f033d317b_add_meta_data_to_batch_processing.py:21]: May need some changes here to align with [others](https://github.com/bcgov/lear/blob/main/legal-api/migrations/versions/9c95a9b28eb3_adding_filing_meta_data.py#L20).

Comment (AimeeGao) [reply legal-api/migrations/versions/ef8f033d317b_add_meta_data_to_batch_processing.py:21]: Thank you for the feedback, @chenhongjing.
I will update the migration script accordingly.

**Review by chenhongjing — APPROVED** [review]
LGTM 😸  Please wait for other's approval. When everything is ready, remember to do "Squash and merge".

**Comment by AimeeGao** [comment]
> LGTM 😸 Please wait for other's approval. When everything is ready, remember to do "Squash and merge".

**Review by leodube-aot — APPROVED** [review]

**Review by argush3 — APPROVED** [review]

**Pull request merged**